R/descriptives.R

```diff
@@ -486,8 +488,10 @@ Descriptives <- function(jaspResults, dataset, options) {
   if (options$mode)                           stats$addColumnInfo(name="Mode",                        title=gettext("Mode"),                    type="number")
   if (options$median)                         stats$addColumnInfo(name="Median",                      title=gettext("Median"),                  type="number")
   if (options$mean)                           stats$addColumnInfo(name="Mean",                        title=gettext("Mean"), 				            type="number")
-  if (options$seMean)              stats$addColumnInfo(name="Std. Error of Mean",          title=gettext("Std. Error of Mean"),      type="number")
-  if (options$sd)              stats$addColumnInfo(name="Std. Deviation",              title=gettext("Std. Deviation"),          type="number")
+  if (options$seMean)                         stats$addColumnInfo(name="Std. Error of Mean",          title=gettext("Std. Error of Mean"),      type="number")
+  if (options$meanCi) {                       stats$addColumnInfo(name="CIUB",                        title=gettext("Upper"),                   type="number", overtitle = gettextf("%s %% Confidence Interval", formattedCiPercent))
+                                              stats$addColumnInfo(name="CILB",                        title=gettext("Lower"),                   type="number", overtitle = gettextf("%s %% Confidence Interval", formattedCiPercent)) }
```

Comment:
```suggestion
  if (options$meanCi) {                       stats$addColumnInfo(name="CIUB",                        title=gettext("Upper"),                   type="number", overtitle = gettextf("%s%% Confidence Interval", formattedCiPercent))
                                              stats$addColumnInfo(name="CILB",                        title=gettext("Lower"),                   type="number", overtitle = gettextf("%s%% Confidence Interval", formattedCiPercent)) }
```
because now there is
![image](https://user-images.githubusercontent.com/21319932/196898340-f51dfa5b-c1da-4f64-8139-9c0255691aa1.png)

while other analyses do not have the space:
![image](https://user-images.githubusercontent.com/21319932/196898438-a395e601-9e5a-454a-ad3a-670eeadfdef0.png)

Reply:
> Since this is entirely subjective it might be that I am the only one that feels this way 😅 @EJWagenmakers & @Kucharssim any other suggestions on the look and feel?

I agree completely with your suggestions.

Reply:
I also prefer this -- will change!